bower.json

```diff
@@ -27,13 +27,14 @@
     "slickgrid": "~2.1.0",
     "reconnectingWebsocket": "#07669519f8",
     "dropzone": "https://github.com/sloria/dropzone.git#accept-directory",
-    "treebeard": "https://github.com/caneruguz/treebeard.git#4112b22be4333c9fce35249131632b6b2a37bb87",
+    "treebeard": "https://github.com/caneruguz/treebeard.git#e22caee9ebf91cc00f73e3912883d2d5c09e1300",
     "jquery.cookie": "~1.4.1",
     "hgrid": "~0.2.10",
     "uri.js": "~1.14.1",
     "xhook": "~1.3.0",
     "styles": "https://github.com/lyndsysimon/styles.git#4db9c8e2f5932cad9f23ad3579d1a82c691faf12",
     "locales": "https://github.com/citation-style-language/locales.git#d2b612f8a6f764cbd66e67238636fac3888a7736",
-    "c3": "~0.4.9"
+    "c3": "~0.4.9",
+    "jsTimezoneDetect": "https://cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.js"
```

Comment:
@sloria and I tried to get the library directly from the repo, but it didn't work for some reason